api/scpca_portal/management/commands/load_data.py

```diff
@@ -203,3 +208,11 @@ def load_data_from_s3(
                     settings.AWS_S3_BUCKET_NAME,
                     computed_file.s3_key,
                 )
+
+        if cleanup_output_data:
```

Comment:
if you add the parameter to allow for overriding:
```suggestion
        if cleanup_output_data or default_cleanup_output_data:
```